### DISCUSSION TITLE
Re-open closed buffers

### DISCUSSION THREAD

**Discussion by thomasschafer** [helix-editor/helix#10813] [Ideas]
It would be useful to be able to re-open closed buffers, either by keeping closed buffers in the jumplist or by adding a command. I am happy to work on the PR, but I'd be interested to know if others have thoughts or preferred methods of implementing this before I begin working on it?

**Comment by markstos** [comment]
I envision following the Recent Files spec in terms of tracking recent files:

https://www.freedesktop.org/wiki/Specifications/recent-file-spec/

Then, have an "Open Recent" picker just as exists in many GUI file menus.

**Reply by thomasschafer** [reply]
Are you thinking this would be written to disk to persist after closing? Or just kept in memory e.g. in the `View` struct?

**Reply by markstos** [reply]
The "recent file spec" requires updating an XML file. Personally, I would update the file as soon as the file is considered "recently  opened" so that the list could persist through Helix closing expectedly unexpectedly. 

I've not a Helix developer, just another user with my own idea. 😀
